docs/config.md

```diff
@@ -55,6 +55,35 @@ for more details.
 | `HttpClient` | [System.Net.Http.HttpClient](https://docs.microsoft.com/dotnet/api/system.net.http.httpclient) and [System.Net.HttpWebRequest](https://docs.microsoft.com/dotnet/api/system.net.httpwebrequest) | * | source |
 | `NetRuntime` | [OpenTelemetry.Instrumentation.Runtime](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Runtime) | * | source |
 
+## Logs
+
+Logs from ASP.NET Core apps could be captured using two options. First option is
+to use HostingStartup and the second option is to rely on CLR Profiler
+Instrumentation. Recommended option for ASP.NET Core is to use HostingStartup.
+For other .NET Core apps logs could be collected only through .NET CLR Profiler
+Instrumentation.
+
+### Enable Logs using HostingStartup
+
+| Environment variable | Required value |
+|-|-|
+| `ASPNETCORE_HOSTINGSTARTUPASSEMBLIES` | `OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper` |
```

Comment:
I see this as  "source instrumentation" for `ILogger` that requires additionally setting `ASPNETCORE_HOSTINGSTARTUPASSEMBLIES`.

Maybe we could try to describe it as such in a `Instrumented logging libraries and frameworks` section?